CONTRIBUTING.md

```diff
@@ -22,6 +22,15 @@ You can run the CLI commands files directly with `go run`. All commands and subc
 go run ./cmd/rhoas kafka create "test" --provider=aws --region=us-east-1
 ```
 
+### Development features
+
+To enable development and early access commands please set environment variable in your terminal:
+
+```bash
+RHOAS_DEV=true
```

Comment:
```suggestion
export RHOAS_DEV=true
```

The help text will appear and completions will work only when RHOAS_DEV is exported.